troi/patches/daily_jams.py

```diff
@@ -19,20 +19,21 @@ class DailyJamsElement(Element):
         Split weekly recommended recordings into 7 sets, one for each day of the week.
     '''
 
-    def __init__(self, recs, day):
+    def __init__(self, recs, user, day):
         Element.__init__(self)
         self.recs = recs
         self.day = day
+        self.user = user
```

Comment:
small thing, can we order the assignments of these in the same order as they are in the function definition?